### PR TITLE
Add check-bgp-summary-netconf.rule

### PR DESCRIPTION
**Pull request by bmadhavi74**

### CODE REVIEW

File: community_supplied/Protocols/Bgp/check-bgp-summary-netconf.rule [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+                sensor bgp-sensor {
+                    path peer-state;
+                }
+                type string;
+                description "Neighbor state";
+            }
+            /*
+            * Anomaly detection logic.
+            */
+            trigger bgp-neighbor {
```
Comment (gvarao) [inline community_supplied/Protocols/Bgp/check-bgp-summary-netconf.rule:83]: change the trigger name to "bgp-neighbor-existence"

File: community_supplied/Protocols/Bgp/check-bgp-summary-netconf.rule [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+                */
+                term neighbor-is-configured {
+                    when {
+                        equal-to "$neighbor-check" 1;
+                        matches-with "$peer-state" Established;
+                    }
+                    then {
+                        status {
+                            color green;
+                            message "Neighbor address ($peer-address) is configured. Neighbor state is Established.";
```
Comment (gvarao) [inline community_supplied/Protocols/Bgp/check-bgp-summary-netconf.rule:100]: change from "Neighbor address ($peer-address) is configured. Neighbor state is Established." to
"Neighbor address ($peer-address) is configured. Neighbor state is $peer-state."

File: community_supplied/Protocols/Bgp/check-bgp-summary-netconf.rule [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+                */
+                term neighbor-is-configured-and-state-is-not-equal-to-established {
+                    when {
+                        equal-to "$neighbor-check" 1;
+                        does-not-match-with "$peer-state" Established;
+                    }
+                    then {
+                        status {
+                            color yellow;
+                            message "Neighbor address ($peer-address) is configured. Neighbor state ($peer-state) is not Established.";
```
Comment (gvarao) [inline community_supplied/Protocols/Bgp/check-bgp-summary-netconf.rule:117]: change message to"Neighbor address ($peer-address) is configured. Neighbor state is $peer-state."

**Review by gvarao — APPROVED** [review]

**Review by gvarao — CHANGES_REQUESTED** [review]
please do changes

**Comment by bmadhavi74** [comment]
Made the required changes and committed.

**Review by gvarao — APPROVED** [review]

**Pull request merged**